%YAML:1.0
---
image_width: 1280
image_height: 1024
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [8.6246419890844390e+02, 0.000000, 6.1909999588631808e+02,0, 8.6387391749562971e+02, 5.3370471772102508e+02, 0.000000, 0.000000, 1.000000]
distortion_coefficients: !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [-9.2914955899754972e-02,8.4684588219656964e-02,3.0363127634307251e-05,1.4757078266352711e-040,0]
